examples/spi/host/src/host.c

```diff
@@ -31,27 +37,36 @@ int main(void)
   }
 
   printf("started\n");
+  printf("using payload of size %d\n", PAYLOAD_LEN);
 
   for (i = 0; i < 4; i++) {
-    payload[0] = i;
-    if (control_write_command(RESOURCE_ID, CONTROL_CMD_SET_WRITE(0), payload, 1) != CONTROL_SUCCESS) {
+    for (int j = 0; j < PAYLOAD_LEN; j++) {
+      payload[j] = i+j;
+
+    }
+
+    if (control_write_command(RESOURCE_ID, CONTROL_CMD_SET_WRITE(0), payload, PAYLOAD_LEN) != CONTROL_SUCCESS) {
       printf("control write command failed\n");
       exit(1);
     }
 
     pause_short();
```

Comment:
pause short? does this mean SPI doesnt work if you don't pause short?

Reply:
It should work, I think is there for debugging purposes only.

Reply:
it would be better to delete it if it is not required or add a comment. As it stands this example makes it look like pauses are required, and the length of the pause is not clear